tests/e2e/templates/velero_bsl_template.tmpl

```diff
@@ -0,0 +1,27 @@
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+ namespace: openshift-adp
+ name: {{ .VeleroName }}
```

Comment:
Better to use DpaName or something to make it more generic

Reply:
Done